src/11_6_2021_RcppClock_benchmarking_Rcpp_code.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Benchmarking Rcpp code with RcppClock"
+author: "Zach DeBruine"
+license: GPL (>= 2)
+tags: Rcpp benchmarking RcppClock

```

Comment:
Please look at existing tags and adjust.  Having a tag `Rcpp` requires too much recursion, so that is a wee bit redundant.  Adding the name of each package is a bit much too (think of tags as 'dimension reductions' _i.e._ keywords, not as index links.  The current list is

```sh
edd@rob:~/git/rcpp-gallery(gh-pages)$ ls tags/
armadillo  c++14          eigen        gsl               mlpack    parallel   s4             string
basics     c++17          environment  integer64         modeling  python     serialization  sugar
benchmark  cube           factor       interrupt         modules   r6         simd           vector
bigmemory  dataframe      featured     machine_learning  nanotime  recursion  simulation     xts
boost      data.table     function     macros            nullable  rmath      sparse
c++11      documentation  gpu          matrix            openmp    rng        stl
edd@rob:~/git/rcpp-gallery(gh-pages)$ 
```

so a single `benchmark` seems apt.

Reply:
tagged with `benchmark` only.